bot.ml

```diff
@@ -545,7 +541,7 @@ let job_action json =
     |> Lwt.async
   else if String.equal build_status "success" then (
     (fun () ->
-      get_status_check ~repo_full_name ~commit ~context
+      get_status_check ~repo_full_name ~commit ~context ~header_list:gitlab_header
```

Comment:
Here you put `gitlab_header` but actually (as explained above), this is a call to the GitHub API.